lib/mox.ex

```diff
@@ -524,7 +524,9 @@ defmodule Mox do
 
     for behaviour <- behaviours,
         {fun, arity} <- behaviour.behaviour_info(:callbacks) do
-      stub(mock, fun, :erlang.make_fun(module, fun, arity))
+      if function_exported?(mock, fun, arity) do
```

Comment:
We should only skip if they are optional though. Otherwise we can stop raising for functions that are required .

Reply:
I'm not sure because this checks if the mock function had been defined on the mock. So when you use `skip_optional_callbacks` with a specific set of callbacks it still works correctly. 

Just checking if it's optional wouldn't cover that, right?

Reply:
I see, I misunderstood. Looks good as is then!